bin/start-collectd

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+: ${GRAPHITE_HOST:=}
+: ${GRAPHITE_PORT:="2003"}
+
+set -eu
+
+CONF_TEMPLATE=/etc/collectd/collectd.conf.tpl
+CONF_DEST=/etc/collectd/collectd.conf
+
+HOSTNAME=$(wget -O - --tries 1 --timeout 0.2 http://169.254.169.254/1.0/meta-data/instance-id 2>/dev/null || hostname)
+
+if [ -n "$GRAPHITE_HOST" ]; then
+    sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+        -e "s/{{ graphite_host }}/$GRAPHITE_HOST/g" \
+        -e "s/{{ graphite_port }}/$GRAPHITE_PORT/g" "$CONF_TEMPLATE"
+else
+  sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+      -e "s/LoadPlugin write_graphite/LoadPlugin write_log/g" \
```

Comment:
@nickstenning try this one out locally and let me know if you like it. The problem with collectd is that if there is no output configured it will send a bunch of error messages on every interval. This will just use the `write_log` output which will write all the metrics to stdout, which is a bit annoying as well.